utils/utils.js

```diff
@@ -98,10 +162,17 @@ const utils = {
       let loc = (message.guild ? `${message.guild?.name} > ${message.channel?.name}` : "DM");
       console.error(`Interaction by ${message.user.username} in ${loc}`);
 
-      message[(message.deferred ? "editReply" : "reply")]({content: "I've run into an error. I've let my devs know.", ephemeral: true}).catch(utils.noop);
+      message[((message.deferred || message.replied) ? "editReply" : "reply")]({content: "I've run into an error. I've let my devs know.", ephemeral: true}).catch(utils.noop);
```

Comment:
This line is a bugfix - `message.deferred` does not get set when an interaction is replied to.